### DISCUSSION TITLE
convert schema to jsonString and update another schema

### DISCUSSION THREAD

**Discussion by subramanian0331** [cue-lang/cue#2103] [Q&A]
i want to update a output schema with a json string of one of the other schema

#jsonData:{
name: string
place: string
}

#output:{
value : string    <-------- i want to serialize jsonData as string and place it here 
}

**Comment by subramanian0331** [comment]
anyone ? help :)

**Comment by mvdan** [comment]
You can use https://pkg.go.dev/cuelang.org/go/pkg/encoding/json: https://tip.cuelang.org/play/?id=-uJUQeRm7CU#cue@export@json
